apps/alchemist/test/integration/alchemist_test.exs

```diff
@@ -56,7 +56,7 @@ defmodule AlchemistTest do
     ]
 
     input_topic = "#{input_topic_prefix()}-#{ingestion.id}"
-    output_topic = "#{output_topic_prefix()}-#{ingestion.id}"
+    output_topic = "#{output_topic_prefix()}-#{dataset.id}"
```

Comment:
oh great change here, this is much more system accurate since the output is a dataset. thank you 👍